db/CSV_Writers/machineRunner.js

```diff
@@ -0,0 +1,18 @@
+const { userProvisions: uP, reviewProvisions: rP } = require('./config/provisions.js');
+const client = require('./config/PGIndex.js');
+
+(async () => {
+  try {
+    await client.connect();
+    await [uP, rP].reduce(async (count, provisions) => {
+      await count;
```

Comment:
what is count? i see youre awaiting count. i thought that would imply some sort of asynchronous function invocation but it doesnt look like count is being invoke.

Reply:
Yeah I could definitely look at other async loops or just go back to two try/catch statements. count isn't anything in this context, its just because in reduce you have to return a value to the accumulator, and because its async reduce that returned value is a promise, from there, I believe that awaiting that promise ensures that each execution is handled sequentially.